src/config/wallets.ts

```diff
@@ -111,6 +114,15 @@ export const supportWalletObj = {
     isSupportBrowserExtension: true,
     isSupportMobileApp: false,
   },
+  [SupportWallet.Metadot]: {
+    img: require('/src/assets/img/logo-metadot.png'),
+    name: 'Metadot Wallet',
```

Comment:
How about changing it to 'Metadot'? So the UI doesn't need to break into 2 lines.

![image](https://user-images.githubusercontent.com/92044428/157504712-d5b656fa-eeea-4764-a4a0-8f68b2cbfaa8.png)


![image](https://user-images.githubusercontent.com/92044428/157504667-4f6117b9-c773-4126-bc0c-86738e7eca26.png)

Reply:
I have done the requested changes, you can check it.
Also I wasn't able to re-produce the errors that are showing in your console, might be the issue of different wallet extensions in your browser but let me know if that is not the case.

Reply:
Hi @bobo-k2 
Could you help check on your end too, please?

https://github.com/AstarNetwork/astar-apps/pull/277#issuecomment-1063190263
https://github.com/AstarNetwork/astar-apps/pull/277#issuecomment-1063214269

Reply:
okay @impelcrypto, I was able to re-produce the errors from your console it is because of running the polkadot.js wallet and Metadot wallet at the same time, we are working to resolve this. In the mean time you can disable the polkadot.js wallet and test the Metadot wallet.

Reply:
![image](https://user-images.githubusercontent.com/8452361/157650677-29e86f9c-61bb-4e8a-8ee3-3696e8f23f22.png)
I got stuck while setting up test account @ahsanahmed321 please advise

Reply:
Yeah @bobo-k2 this is a known bug that we have fixed on our side as the error is not shown correctly. Try using the password according to standards mentioned below the password input box (e.g use lower case, upper case, number & special character in your password.)

Reply:
@impelcrypto @bobo-k2 were you guys able to test it out?

Reply:
@khuzama98 

> https://github.com/AstarNetwork/astar-apps/pull/277#discussion_r823502440

Have you fixed the error?

Reply:
Also, please fix [comment]( https://github.com/AstarNetwork/astar-apps/pull/277#issuecomment-1063190263)

Reply:
Yes @impelcrypto @bobo-k2 , we're sending a new build to @hoonsubin .

Reply:
Guys @bobo-k2 @impelcrypto did you guys get the new build?

Reply:
We received the new build. Please give us some time to finish the review.